modules/server/src/main/scala/navigate/server/tcs/M1GuideConfig.scala

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package navigate.server.tcs
+
+import cats.{Show, Eq}
+import cats.syntax.all.*
+import navigate.model.enums.M1Source
+
+/** Data type for M1 guide config. */
+sealed trait M1GuideConfig extends Product with Serializable {
+  def uses(s: M1Source): Boolean
+}
+
+object M1GuideConfig {
+  case object M1GuideOff                       extends M1GuideConfig {
+    override def uses(s: M1Source): Boolean = false
+  }
+  final case class M1GuideOn(source: M1Source) extends M1GuideConfig {
+    override def uses(s: M1Source): Boolean = source === s
+  }
+
+  object M1GuideOn {
+    implicit val showM1GuideOn: Show[M1GuideOn] = Show.fromToString
```

Comment:
`given`?

Reply:
That happens when you copy code from an old project.